### PR TITLE
LPD-15900: Change ClayLabel displayType depending on the httpMethod name

### PR DESCRIPTION
**Pull request by sergiojimcos**
**What's new?**
- Fix displayType depending on the `httpMethod` value of the API Endpoint

**Before the PR**
Screenshot from 2024-01-30 16-34-51

**After PR**
 
Screenshot from 2024-01-30 16-34-18

**JIRA Related Ticket**
https://liferay.atlassian.net/browse/LPD-15900

### CODE REVIEW

**Comment by sergiojimcos** [comment]
ci:test:sf

File: modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/FDS/fdsUtils/fdsRenderers.tsx [hunk 1]
```diff
@@ -16,7 +16,18 @@ export function itemMethodRenderer({
 }: {
 	itemData: {httpMethod: {name: string}};
 }) {
-	return <ClayLabel displayType="info">{itemData.httpMethod.name}</ClayLabel>;
+	if (itemData.httpMethod.name === 'post') {
+		return (
+			<ClayLabel displayType="success">
```
Comment (jaimelr10) [inline modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/FDS/fdsUtils/fdsRenderers.tsx:21]: How do you see creating a function here to set the displayType?

Something like `<ClayLabel displayType={getDisplayType()}>`

This getDisplayType() returning "success" or "info" depending on the httpMethod.

**Review by jaimelr10 — CHANGES_REQUESTED** [review]
How do you see my comment? :smile:

File: modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/FDS/fdsUtils/fdsRenderers.tsx (hunk 1 above)
Comment (sergiojimcos) [reply modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/FDS/fdsUtils/fdsRenderers.tsx:21]: Make a lot of sense @jaimelr10! :smile:

**Comment by sergiojimcos** [comment]
ci:forward:force

**Pull request closed without merging**